content/monitors/downtimes.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Downtimes
 kind: documentation
+description: "Schedule downtimes so that your Datadog monitors do not alert at specific times when you do not want them to."
```

Comment:
Schedule downtimes to ensure your Datadog monitors do not alert during specific time periods.